doc/z-chap15.xml

```diff
@@ -66,5 +66,121 @@
     <#Include Label = "IsMonogenicInverseMonoid">
 
   </Section>
- 
+
+  <!--**********************************************************************-->
+  <!--**********************************************************************-->
+
+  <Section Label = "McAlister triple semigroups">
```

Comment:
@james-d-mitchell Should this be its own chapter? Is this the right place for this doc? If so, should the title of this chapter be changed from `15 Properties and attributes specific to inverse semigroups`?